src/lib/translator.ts

```diff
@@ -53,9 +49,14 @@ export class Translator implements TranslatorInterface {
         //   if none found find by short locale like "de"
         //   if still none found search by the fallback which is by default "en"
         let translated = "";
-        [this.long(), this.short(), this._options.fallback].find(
-            (tag) => !!pattern.find((pat) => !!(translated = this._resources[`${tag}.${pat}`])),
-        );
+        [this.long(), this.short(), this._options.fallback].some((tag) => {
```

Comment:
it may be better to loop first through long() and short() and if then nothing is found to add the pattern + try to find it within the fallback resource.